src/utils/form.js

```diff
@@ -98,7 +98,7 @@
                         return tpl_select_option({
                             'value': value,
                             'label': option.getAttribute('label'),
-                            'selected': values.includes(value),
```

Comment:
You can use `_.includes` here instead. 

Also, I forgot to ask in the previous PR, please add en entry for the original fix in `CHANGES.md`.

Thanks